example/src/main/scala/example/AuthenticationClient.scala

```diff
@@ -0,0 +1,21 @@
+package example
+
+import zhttp.http.Headers
+import zhttp.service.{ChannelFactory, Client, EventLoopGroup}
+import zio.{App, ExitCode, URIO}
+
+object AuthenticationClient extends App {
+  // start AuthenticationServer before running this client
+  val url = "http://localhost:8090"
+  val env = ChannelFactory.auto ++ EventLoopGroup.auto()
+
+  val program = for {
+    token    <- Client.request(s"${url}/login/username/emanresu").flatMap(_.bodyAsString)
```

Comment:
explain why the first request is done, wha tis the expected response. I think that should not assume that users knows the Bearer flow.

Reply:
Done.